sources/frontend.c

```diff
@@ -752,11 +752,19 @@ static od_frontend_status_t od_frontend_remote_server(od_relay_t *relay,
 		break;
 	case KIWI_BE_COPY_IN_RESPONSE:
 	case KIWI_BE_COPY_OUT_RESPONSE:
-		server->is_copy = 1;
+		server->in_out_response_received++;
 		break;
 	case KIWI_BE_COPY_DONE:
-		server->is_copy = 0;
+		/* should go after copy out
+		* states that backend copy ended
+		*/
+		server->done_fail_response_received++;
```

Comment:
we need a name, that indicated that this is a number of copy dones + FE copy fails